src/components/animations/Animation07/Animation07.module.css

```
@@ -0,0 +1,366 @@
+  position: relative;
+  max-width: 90%;
+}
+
+/* Bouncing Text Styles */
+.bouncingText {
+  font-size: clamp(3rem, 8vw, 8rem);
+  font-weight: 900;
+  letter-spacing: 0.1em;
+  margin-bottom: 2rem;
+  line-height: 1.2;
+  margin: 0;
```

Biome (2.1.2)

[error] Unexpected shorthand property margin after margin-bottom



(lint/suspicious/noShorthandPropertyOverrides)